### ISSUE TITLE
Possibility to change auto complete shortcut to Tab instead of arrow key

### ISSUE THREAD

**Issue by wazeerc** [zsh-users/zsh-autosuggestions#656]
Just wondering if there's the possibility to re bind the auto-complete button (which is the right arrow key by default) to **Tab**.

**Comment by ericfreese** [comment]
Yeah take a look at the bottom of this section of the Readme: https://github.com/zsh-users/zsh-autosuggestions#key-bindings

You can use `^I` to represent tab when calling bindkey

**Comment by Rhahkeem** [comment]
> Yeah take a look at the bottom of this section of the Readme: https://github.com/zsh-users/zsh-autosuggestions#key-bindings
> 
> You can use `^I` to represent tab when calling bindkey
$ZSH_CUSTOM`  directory?

**Comment by ngshijun** [comment]
Hi, is there any precise guide on how to change the autocomplete key binding from right arrow to tab?

**Comment by danigiorgio** [comment]
> > Yeah take a look at the bottom of this section of the Readme: https://github.com/zsh-users/zsh-autosuggestions#key-bindings
> > You can use `^I` to represent tab when calling bindkey
> 
> where would you put this binding? Is it in a specific file or just in an `overrides.zsh in the $ZSH_CUSTOM` directory?

Put it in ./zshrc (open it by running `open ~/.zshrc`)

it would be like this, make sure it's below plugins.

```
plugins=(
 zsh-autosuggestions
)

bindkey '^I' autosuggest-accept
```

**Comment by 0xTadash1** [comment]
`bindkey '^I' autosuggest-accept`

With this method, the normal <kbd>Tab</kbd> completion is disabled.

Ideally, it would be great to insert the zsh-autosuggestions result at the top of the <kbd>Tab</kbd> completion list.
